src/czar/ActiveWorker.cc

```diff
@@ -191,7 +192,7 @@ void ActiveWorker::_sendStatusMsg(protojson::WorkerContactInfo::Ptr const& wInf,
                 czar::Czar::getCzar()->killIncompleteUbjerJobsOn(wInf->wId);
             }
         } else {
-            LOGS(_log, LOG_LVL_ERROR, cName(__func__) << " transmit failure response success=0 " << response);
+            LOGS(_log, LOG_LVL_ERROR, cName(__func__) << " transmit failure " << *respMsg);
```

Comment:
For better readability of the logged messages, it may make sense to add a separator `:` between:
```
" transmit failure: " << *respMsg
```